### ISSUE TITLE
Place to add new model features?

### ISSUE THREAD

**Issue by mdekauwe** [PecanProject/pecan#1194]
Is there a place to highlight new model features anywhere? I've just pushed SPA hydraulics into the main branch of GDAY (although I'm still probably the only person on the planet who knows how to use this functionality at the moment). Similarly, I will probably push our P-cycle to the main branch this week as well.

I realise the GDAY implementation is probably not functional, but I wondered if there is a top-level location where one could advertise new features to the PECAN community? Otherwise it strikes me you'd need to be up to date with a specific models developments which might not be the case.

**Comment by tonygardella** [comment]
I think a good place for that would be in the model specific pages of the [GDAY page of our gitbook](https://github.com/PecanProject/pecan.book/blob/master/models/gday.md).  Our intention with these pages is to not only list basic information about each model, but to also summarize key features of models.

On that note, I need to get back into gear and finish off the GDAY coupling scripts. I'll be in touch with you about that.

**Comment by mdietze** [comment]
@mdekauwe good catch that we don't have any official policy/mechanism for this. We'll add this to our weekly agenda.

The other things that I'd suggest:
1) Let us know (especially @tonygardella and @robkooper) when there's a new version of your model so we know to update our own machines and update the VM with the new version in the next release
2) You can also add this info directly to the new [changelog](https://github.com/PecanProject/pecan/blob/master/CHANGELOG.md), which will be the basis of any release notes when we announce new PEcAn versions. A Pull Request to the change log might also be the most efficient way of letting the PEcAn developer community know about the new version.
3) If you tweet about any new features, make sure to tag pecan, so we can retweet to folks interested in the project. If/when we set up a user's email list, we can also announce changes there as well.

**Comment by dlebauer** [comment]
@mdekauwe vignettes (in the model package) could be another place to show off new features - e.g. a comparison of model skill with / without the new feature would be cool.
